site/content/docs/5.3/components/local-navigation.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+title: Local navigation
+description: Use local navigation to add a navigation that will nicely wrap on small viewports.
+group: components
+aliases:
+  - "/docs/components/local-navigation/"
+toc: true
+added: "5.3"
+---
+
+## Example
+
+Resize your browser to see the behavior of this component.
+
+{{< callout info >}}
+Since the component is supposed to be rendered on a full size page and not inside an example with padding, there might be some rendering issues in mobile viewport.
```

Comment:
```suggestion
Since the component is supposed to be rendered on a full-size page, there are some slight rendering issues in the mobile viewport in the context of this documentation page.
```